src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2098,6 +2098,17 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
             $container->getDefinition('messenger.transport.beanstalkd.factory')->addTag('messenger.transport_factory');
         }
 
+        if ($this->hasConsole()) {
+            $container->getDefinition('messenger.listener.stop_worker_signals_listener')->clearTag('kernel.event_subscriber');
```

Comment:
It'd probably be better to remove the definition itself, but that'd be a BC break.

Reply:
The `messenger.listener.stop_worker_signals_listener` service may not exist in case the installed version of the Messenger component is >= 7.0.

Reply:
Yes, I can update #51093 once this is merged.

Reply:
#51093 has been merged now.